cognite/neat/rules/models/_rules/_types/_base.py

```diff
@@ -222,6 +227,60 @@ def as_id(
         return ViewId(space=space, external_id=external_id, version=version)
 
 
+class ViewPropEntity(ViewEntity):
+    type_: ClassVar[EntityTypes] = EntityTypes.view_prop
+    property_: str | None = None
```

Comment:
A bit ugly way to store symmetric property, we can fix this later. 

But rename `property_` to `symmetric_property` since If I am not mistaken you are storing reverse property here, not actual property that points to this View.